fern/products/sdks/overview/typescript/dynamic-authentication.mdx

```diff
@@ -37,6 +37,31 @@ groups:
           allowCustomFetcher: true
 ```
 
+### Add runtime dependency via packageJson
+
+Add the `jsonwebtoken` dependency using the `packageJson` config option so it gets merged into the generated package.json:
+
+```yaml title="generators.yml"
+default-group: local
+groups:
+  local:
+    generators:
+      - name: fernapi/fern-typescript-node-sdk
+        version: 0.x.x
+        output:
+          location: local-file-system
+          path: ../generated/typescript
```

Comment:
omit parts of the generators.yml that are not relevant to this change. 
Also, it's fernapi/fern-typescript-sdk, not fernapi/fern-typescript-node-sdk